### DISCUSSION TITLE
post/adding-a-comment-section-to-your-blog-posts-or-website-using-giscus-in-next-js-what-is-giscus

### DISCUSSION THREAD

**Discussion by giscus** [madinkum/next.js-sanity-blog#5] [Q&A]
# post/adding-a-comment-section-to-your-blog-posts-or-website-using-giscus-in-next-js-what-is-giscus

http://localhost:3000/post/adding-a-comment-section-to-your-blog-posts-or-website-using-giscus-in-next-js-what-is-giscus

**Comment by madinkum** [comment]
This is awesome

**Reply by TechMachire** [reply]
see if this works

**Comment by TechMachire** [comment]
hmmm

**Reply by TechMachire** [reply]
I need to be able to delete my comments
